modules/pubmaticBidAdapter.js

```diff
@@ -335,9 +621,80 @@ function _handleEids(payload) {
   }
 }
 
+function _parseNativeResponse(bid, newBid) {
+  newBid.mediaType = 'native';
+  newBid.native = {};
+  if (bid.hasOwnProperty('adm')) {
+    var adm = '';
+    try {
+      adm = JSON.parse(bid.adm.replace(/\\/g, ''));
+    } catch (ex) {
+      utils.logWarn(BIDDER_CODE + ' Error: Cannot parse native reponse for ad slot: ' + bid.params.adSlot);
+      return;
+    }
+    if (adm && adm.native && adm.native.assets && adm.native.assets.length > 0) {
+      for (let i = 0, len = adm.native.assets.length; i < len; i++) {
+        switch (adm.native.assets[i].id) {
+          case NATIVE_ASSET_ID.TITLE:
+            newBid.native.title = adm.native.assets[i].title && adm.native.assets[i].title.text;
+            break;
+          case NATIVE_ASSET_ID.IMAGE:
+            newBid.native.image = {
```

Comment:
In this assignment, if we could push name of the key on the left-hand-side into a variable then we can reuse same statements for case of NATIVE_ASSET_ID.ICON

Reply:
Should i maintain another constant object to maintain keys to achieve the same? or any other alternative ?